.pre-commit-config.yaml

```diff
@@ -85,7 +85,7 @@ repos:
           - "#"
       - id: insert-license
         name: Insert license headers (XML-style comments)
-        files: '\.(?:html|md|xml)$'
+        files: '\.(?:html|md|xml|jinja)$'
```

Comment:
Jinja doesn't imply XML-style. We should use `{# jinja-style comments #}` instead.